Sofa/Component/Topology/Mapping/src/sofa/component/topology/mapping/Triangle2EdgeTopologicalMapping.cpp

```diff
@@ -66,25 +66,38 @@ Triangle2EdgeTopologicalMapping::~Triangle2EdgeTopologicalMapping()
 
 void Triangle2EdgeTopologicalMapping::init()
 {
-    // recheck models
     bool modelsOk = true;
+
+    // Check input topology
     if (!fromModel)
     {
         msg_error() << "Pointer to input topology is invalid.";
         modelsOk = false;
     }
+    else
+    {
+        // Making sure the input topology includes triangle elements
+        if (fromModel.get()->getNbTriangles() != 0)
```

Comment:
This could still be valid if we start from an empty mesh and we add progressively triangles with a mapping on the edges... btw I thought I had a test on that.